src/app/asset-page/agree-modal/agree-modal.component.ts

```diff
@@ -30,20 +30,21 @@ export class AgreeModalComponent implements OnInit {
   @Input()
   isMSAgent: boolean
 
-  @ViewChild("modal", {read: ElementRef}) modalElement: ElementRef;
+  @ViewChild("agreeModalTitle", {read: ElementRef}) modalTitleElement: ElementRef;
 
   constructor(
     private _auth: AuthService,
     private _dom: DomUtilityService
   ) { }
 
-  ngOnInit() {
-    // Set focus to the modal to make the links in the modal first thing to tab for accessibility
-    // TO-DO: Only reference document client-side
-    // let htmlelement: HTMLElement = <HTMLElement>this._dom.byId('modal');
-    // htmlelement.focus()
-    if (this.modalElement && this.modalElement.nativeElement){
-      this.modalElement.nativeElement.focus()
+  ngAfterViewInit() {
+    this.startModalFocus()
+  }
+
+  // Set initial focus on the modal Title h4
```

Comment:
Curious if we want to be setting the focus on the h4. Traditionally I think we set focus to the first "interactive" element on the modal. Like in this case I think it would be the terms and conditions link?

Reply:
There used to be the expectation to focus first on the heading so that the screen reader will read out what this modal is. I don't know if we want to change that.

Reply:
Ah I see. If that is the old expected behavior I say we just stick with that